.rubocop.yml

```diff
@@ -10,6 +10,11 @@ Style/Documentation:
 Style/NonNilCheck:
   IncludeSemanticChanges: true
 
+Style/FileName:
+  Exclude:
+  - 'Gemfile.local.rb'
```

Comment:
i tried to use the inherited rubocop file, but then it would have too much offenses, so i suggest we postpone that task.